tendermint/Cargo.toml

```diff
@@ -27,16 +27,17 @@ authors = [
 
 [package.metadata.docs.rs]
 all-features = true
-
-[badges]
-codecov = { repository = "..."}
+rustdoc-args = ["--cfg", "docsrs"]
 
 [dependencies]
 anomaly = "0.2"
 async-trait = "0.1"
 bytes = "0.5"
 chrono = { version = "0.4", features = ["serde"] }
+ed25519 = "1"
+ed25519-dalek = { version = "= 1.0.0-pre.4", features = ["serde"] }
 futures = "0.3"
+k256 = { version = "0.4.2", optional = true, features = ["ecdsa"] }
```

Comment:
Are we dependent on this specific patch version? Or can we loosen the constraint?

Reply:
This does not pin the specific patch version. It's equivalent to "^0.4.2".

Reply:
It could be relaxed to "0.4" without issue. Like @liamsi said, this is a SemVer baseline.

I think I originally went with such a specific version because this patch started as a Signatory upgrade and that was the first version it would've supported, but then I decided to rip Signatory out as it's not useful for this crate anymore and the other crates have subsumed much of its functionality and common signing/verification APIs.